### PR TITLE
Sub category of components - animation components to make it tidier & a component suggestion

### PR DESCRIPTION
**Pull request by mentaman**
Hey I think it'll be better if instead of only the big "components" section that there is now.
Split it to a few components sections.
I started it with animation components and united all of them into the new animation components section.

Advantages:

- Less conflicts. The more splitted the readme. the less chances two different people will change the same line. Because they usually put it in the end of every category. 
- Easier to read. Cleaner and tidier. The components section is too big to actually go through it and to be useful..

Also, I want to suggest the library "react-singular-component", it's not a very popular one, but it is great in my opinion. It gives you a way to easily create a transition between two dom elements of a component.
Useful when:
- You have the same component in a few places of your website, but not at the same time. And when one is gone you want it to go smoothly to the other dom.

This gif demonstrates what it looks like:
alt text

This react component is efficient and customable, and already tried and works on a few products I made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +36,7 @@ A collection of awesome things regarding React ecosystem.
     - [Routing](#routing)
     - [React Projects](#react-projects)
     - [Components](#components)
+    - [Components](#animation-components)
```
Comment (Haroenv) [inline README.md]: this name needs to change :)

File: README.md [hunk 2]
```diff
@@ -604,8 +601,16 @@ A collection of awesome things regarding React ecosystem.
 * [react-location-picker - A react component to pick a location using google maps.](https://github.com/rameshsyn/react-location-picker)
 * [cogo-toast - Beautiful, Zero Configuration, Toast Messages](https://github.com/Cogoport/cogo-toast)
 * [folio - Dynamic Form Built With React & CSS Grid layout](https://github.com/jalal246/folio)
-* [react-loading-skeleton - Make beautiful, animated loading skeletons that automatically adapt to your app.](https://github.com/dvtng/react-loading-skeleton)
 * [material-ui-layout - A declarative Layout component based on Material-UI and Google's Material Design](https://github.com/OrigenStudio/material-ui-layout)  
+*
```
Comment (Haroenv) [inline README.md]: this can just be a newline
```suggestion

```

**Comment by mentaman** [comment]
Oops. my bad. 
Fixed

**Comment by Haroenv** [comment]
Thanks!

**Comment by mentaman** [comment]
Awesome :D

**Comment by Haroenv** [comment]
Don't hesitate to add more things or reorder if you feel like it!

**Pull request merged**